recipes/calcephpy/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "calcephpy" %}
+{% set version = "3.4.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e2f33ccde6f923c42135674442f3055619d3b3c1142a7b041f5c5c9c83bb422e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
+    - setuptools
+    - cython
```

Comment:
python, cython and numpy shouldn't be in the build, only in host. `build` is only for compilers/build-tools whereas `host` are the build-time dependencies.